android-app/app/src/main/java/de/dhbw/ka/se/fibo/ui/adding/AddingFragment.java

```diff
@@ -150,16 +164,11 @@ private void initializeButtons() {
     }
 
     private Cashflow createCashFlow() {
-        boolean isRequiredDataPresent = false;
+        boolean isRequiredDataPresent = checkForRequiredData();
```

Comment:
This line is not needed anymore. You can check it in line 173 with `if (checkForRequiredData())` instead of `if (isRequiredDataPresent)`